components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -308,6 +308,17 @@ message ControlItemRequest {
 	repeated ListFilter filters = 3;
 }
 
+message WaiverData {
+    // String identifying the waived state could be one of: yes, yes_run, no, no_expired
+	string waived_str = 1;
+	// The expiration date for the waiver.. after this date, the control is no longer waived
+	string expiration_date = 2;
+	// The reason for the waiver
+	string justification = 3;
+	// What are the counts for the waived controls across nodes?
```

Comment:
Due to a fun quirk in how our docs tools work, this text will not show up in the UI. The description for this field will come from where the message is defined, here: https://github.com/chef/automate/blob/9af83565eb855e093f455149b2c4a7bb985c379c/components/automate-gateway/api/compliance/reporting/reporting.proto#L404-L405

The convention we have to help demystify this is to use filler text for things like this.
```suggestion
	// Intentionally blank.
```